### PR TITLE
86,118,125: bug tickets

### PR DESCRIPTION
**Pull request by IlliaLop**

### CODE REVIEW

File: frontend/src/app/modules/user-profile/user-profile-page/user-profile.component.ts [hunk 1]
```diff
@@ -143,6 +143,7 @@ export class UserProfileComponent extends BaseComponent implements OnInit, OnDes
                     this.spinner.hide();
                     this.notificationService.info('Names successfully updated');
                     this.initUserNamesForm();
+                    window.location.reload();
```
Comment (serhiiyasenev) [inline frontend/src/app/modules/user-profile/user-profile-page/user-profile.component.ts]: do we really need to reload the whole page or probably just `this.fetchCurrentUser()` or smth like that?

Comment (IlliaLop) [reply frontend/src/app/modules/user-profile/user-profile-page/user-profile.component.ts]: > do we really need to reload the whole page or probably just `this.fetchCurrentUser()` or smth like that?

This function will not update our drop-down. You need to either use the EventService to which our drop-down is subscribed or update our page as I did

**Review by serhiiyasenev — DISMISSED** [review]

File: frontend/src/app/modules/user-profile/user-profile-page/user-profile.component.ts (hunk 1 above)
Comment (Anatolii-Ivanov) [reply frontend/src/app/modules/user-profile/user-profile-page/user-profile.component.ts]: Let's not reload page, please

Comment (IlliaLop) [reply frontend/src/app/modules/user-profile/user-profile-page/user-profile.component.ts]: > Let's not reload page, please

ok. I'll fix it now

**Review by serhiiyasenev — CHANGES_REQUESTED** [review]
GitHub actions

**Review by serhiiyasenev — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**